signalfx_lambda/utils.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+from .version import name, version
+
+fields = {}
+
+def get_fields(context):
+    function_arn = context.invoked_function_arn
+
+    global fields
```

Comment:
mutating objects don't require the global keyword.  There is only one `fields` dictionary that can be modified by anything w/ access to it.